avocado/core/test.py

```diff
@@ -961,15 +957,14 @@ def test(self):
         raise exceptions.NotATestError(e_msg)
 
 
-class SkipTest(Test):
+class FakeTest(Test):
```

Comment:
Actually these are actual tests, so please ignore the first comment. Also no need to change anything in the second comment as I'll be tackling this in my PR so just let me know about the naming suggestion, anyway it's really minor as I'm not native and my perceiving can be wrong.